### DISCUSSION TITLE
CAT issue

### DISCUSSION THREAD

**Discussion by GregTroar** [wavelog/wavelog#1042] [General]
Hello,

Just starting testing WaveLog.
I have an issue with CAT, I installed FLRig on a windows 11 machine and WaveLogGate, I could connect WaveLogGate to Wavelog, the radio is recognized in the hardware and frequency is updated properly.
Same in bandmap/bandlist it works when I select the radio BUT in live QSO the band mode and frequency are not updated.

Did I miss something ?

Thanks

**Comment by 205er** [comment]
![image](https://github.com/user-attachments/assets/0a391932-541f-4488-8793-94bc06d405cc)
Check the second tab - is the TRX set there?

**Comment by HB9HIL** [comment]
As this does not look like a bug, but could help other I moved this one from WavelogGate Repo (Issues) to Wavelog Main Repo (Discussions)

**Comment by GregTroar** [comment]
Yes in station radio is selected, and in Band map the frequency is properly updated while in live QSO it does not work

**Comment by GregTroar** [comment]
![image](https://github.com/user-attachments/assets/848af878-99b8-4d63-ad01-c2502e1ac705)

![image](https://github.com/user-attachments/assets/b3b28131-611a-41c9-aced-7e1d5c3943de)

![image](https://github.com/user-attachments/assets/e49a5e4e-ca16-44e6-a0f0-070da0062ff5)

**Comment by HB9HIL** [comment]
As it seems you are using the wave-flex-integrator, please refer this issue to https://github.com/tnxqso/wave-flex-integrator

**Comment by GregTroar** [comment]
With flrig and the connector it is the same though.
Does work in band map but not in live qso

**Comment by HB9HIL** [comment]
@GregTroar It's difficult to say whats wrong there. Here it works without issues. So we need a little bit more information like which browser do you use, which other applications are running at the same time (maybe try close everything else except WLgate, FLrig and a browser). Describe your workflow, what do you do exactly to reproduce this issue...

**Reply by HB9HIL** [reply]
You could do this by creating an issue https://github.com/wavelog/wavelog/issues/new/choose

**Comment by mccomber** [comment]
I'm experiencing the same problem. I have Flrig and wavelog-gate running and communicating with each other and my Wavelog install.  Hardware Interfaces shows the radio with the correct mode/freq and updates in real time when I adjust the radio.  The Bandmap QSY's my radio when I select a callsign.  But in Live QSO mode, band and freq are incorrect and default to AM/160m/1838.  Yes I have the radio selected in the Station tab.

Install of everything was done today so versions are up to date as of Feb 18th. Flrig and wavelog-gate are running on Win10, and Wavelog is running on an Ubuntu server with PHP 7.4.

I have tried a few browsers and the behaviour is the same in all.

**Reply by mccomber** [reply]
There was a distro LTS release upgrade available so I did, which brought PHP up to 8.1 and the issue is resolved. Looks like it is a 7.4 compatibility problem.

**Reply by mccomber** [reply]
I should at upgrading to PHP 8.1 also fixed two issues with LotW that I had as well (couldn't log in essentially).  I'd say 7.4 is no longer supported lol.  Cheers.

**Reply by int2001** [reply]
if you look into the QSO-View, is there any kind of "banner", after you've chosen the Radio?
Something like this:

<img width="796" alt="image" src="https://github.com/user-attachments/assets/8b5fb383-a94d-48ad-b9cd-bf74f617a38c" />

**Reply by mccomber** [reply]
Yes. The banner is there above Previous Contacts and Mode/Band/Freq are all correctly filled in since upgrading to PHP 8.1.  Back when the server was on PHP 7.4 none of those showed up, even though things worked in the Bandmap.

**Reply by int2001** [reply]
and the banner shows the correct values?!

**Reply by mccomber** [reply]
Yes it does.

**Reply by int2001** [reply]
ok. if you press F12 in your Browser - are there any javascript-errors visible on console?
Try to turn VFO. The Banner should update - pse look out for any errors on the JS-Console while turning VFO

**Comment by mccomber** [comment]
No JS errors. VFO/mode updates are responsive in both the QSO fields and the banner.

**Reply by int2001** [reply]
this is more than strange.
the logic which updates the banner is nearly the same like that which updates the QSO-Form at live-QSO.

<img width="915" alt="image" src="https://github.com/user-attachments/assets/1122dbf5-1945-4206-8c6f-f8a8a9ae0c56" />

So if the right one is filled, the left one should(!) be filled as well.
Can you please post a screenshot of the QSO-View with the Banner at the QSO-Tab (including the left part)

**Reply by mccomber** [reply]
I think you are misunderstanding me.  Since I upgraded my server to PHP 8.1 from PHP 7.4, everything is working as it should.

**Reply by int2001** [reply]
your statement was: CAT doesn't work // Frequency and mode are not at QSO-Form.
i try to debug that with you.

you told us: Banner is there, Cluster works, but Frequency and Mode is not at Form.

Since CAT is working in general (DXCluster, Banner), this is probably no PHP-Problem.

**Reply by mccomber** [reply]
Look at my first post then look at my reply to myself in that post from 17 hours ago.

**Reply by mccomber** [reply]
![Capture](https://github.com/user-attachments/assets/505b6ed8-d5f6-4837-b13b-3849b9994171)

**Reply by int2001** [reply]
> I'm experiencing the same problem. I have Flrig and wavelog-gate running and communicating with each other and my Wavelog install. Hardware Interfaces shows the radio with the correct mode/freq and updates in real time when I adjust the radio. The Bandmap QSY's my radio when I select a callsign. But in Live QSO mode, band and freq are incorrect and default to AM/160m/1838. Yes I have the radio selected in the Station tab.

> Install of everything was done today so versions are up to date as of Feb 18th. Flrig and wavelog-gate are running on Win10, and Wavelog is running on an Ubuntu server with PHP 7.4.

so whats the point here? Perhaps really misunderstood. You state (as i cite you here): "...But in Live QSO mode, band and freq are incorrect...".

**Reply by int2001** [reply]
no problem any longer? right?

**Reply by mccomber** [reply]
Correct. I posted when I had a problem. Then I thought it might be an issue with PHP 7.4 so I upgraded. When that solved the issue I posted that it did so.

**Reply by int2001** [reply]
ok. tnx for clarifying

**Reply by mccomber** [reply]
Thank you too. Despite our initial difficulties understanding each other, I greatly appreciate what you do and those who work on this project.
